installer_scripts/install-yb-voyager

```diff
@@ -6,19 +6,37 @@ export LC_ALL=en_US.UTF-8
 
 set -e
 
-VERSION="latest"
-if [ "$1" != "" ]
-then
-	if [ "$1" == "local" ]
-	then
-		# Install from locally checked out branch.
-		VERSION=local
-	else
-		echo "ERROR: Invalid argument: '$1'"
-		exit 1
-	fi
+OPTS=$(getopt -o v: --long version:,only-pg-support -n 'install-yb-voyager' -- "$@")
+
+if [ $? != 0 ]; then 
+	echo "Unable to process the options. Exiting..." >&2; 
+	exit 1; 
 fi
 
+eval set -- "$OPTS"
+
+VERSION="latest"
+ONLY_PG="false"
+
+while true; do
+	case "$1" in
+		-v | --version ) 
+			if [ "$2" == local ]; 
+			then 
+				# Install from locally checked out branch.
+				VERSION="$2"; shift 2;
+			else
+				echo "ERROR: Invalid version"
+				exit 1
+			fi 
+			;;
+		--only-pg-support ) ONLY_PG="true"; shift ;;
```

Comment:
Move the option processing into separate lines:
```
  --only-pg-support )
    ONLY_PG="true"
    shift
    ;;
```

Reply:
Done